appinfo/app.php

```diff
@@ -1,16 +1,8 @@
 <?php
 
-$l = \OC::$server->getL10N('files_external_ftp');
-
-OC_Mount_Config::registerBackend('\OCA\Files_External_FTP\FTP', [
-	'backend' => (string)$l->t('FTP (Fly)'),
-	'priority' => 100,
-	'configuration' => [
-		'host' => (string)$l->t('hostname'),
-		'username' => (string)$l->t('Username'),
-		'password' => (string)$l->t('Password'),
-		'root' => '&' . $l->t('Remote subfolder'),
-		'ssl' => '!' . $l->t('Secure ftps://'),
-		'port' => '&' . $l->t('Port'),
-	],
-]);
+\OC::$server->getEventDispatcher()->addListener(
+	'OCA\\Files_External::loadAdditionalBackends', function($event) {
```

Comment:
If files_external_ftp is loaded after files_external, the event dispatcher trigger will have already fired, so this listener won't be activated. You probably need to check if files_external is available yet, and if not, then go via the event dispatcher. This is a consequence of not knowing which order apps are loaded in, and not having the core bits of files_external in core.